### DISCUSSION TITLE
Cannot delete category

### DISCUSSION THREAD

**Discussion by infinitepower18** [community/community#3352] [Discussions]
I can't delete the default Show and Tell category. I get this when trying to delete it:

![image](https://user-images.githubusercontent.com/44692189/107511175-5695eb00-6bcf-11eb-933d-07879f33116f.png)

**Comment by PranamBhat** [comment]
This happens when your session is expired or your account has been logged out. Just refresh the page and try again.

**Comment by panzarino** [comment]
I'm also running into this same issue while my session is not expired and my account is logged in. It'll put the discussions into a weird state where discussions that were in the deleted category are transferred to the new category, but the old category still shows up everywhere and discussions cannot be moved into it.

**Comment by JimmyCushnie** [comment]
I am also experiencing this issue.

**Reply by gilgongo** [reply]
I confirm this as well. Recently turned on Discussions, want to remove the Q&A category (which has no posts in it), but I get a unicorn. When I refresh the page, the category is still there.
